pkg/test/server.go

```diff
@@ -78,3 +95,14 @@ type InstallStrategy struct {
 func (s *InstallStrategy) GetCRDs() []*extensionsv1beta1.CustomResourceDefinition {
 	return s.crds
 }
+
+func (te *TestEnvironment) GetKubeCtl() *integration.KubeCtl {
```

Comment:
Where is this used?

Reply:
It has not been used in generated code by far though, user can invoke kubeCtl by this method later.